x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/alert_summary_widget_ui.stories.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AlertsSummaryWidgetUI as Component } from './alert_summary_widget_ui';
+
+export default {
```

Comment:
👍🏻 Very cool adding a Storybook! I did not know how storybook was setup in kibana, but it looks very straightforward to add stories.